int8 set_value
---
int8 ret